index.html

```diff
@@ -28,9 +28,7 @@ <h1>Year progress bar</h1>
 
     <div class="progress" id="progress"></div>
 
-    <h1 id="percent"></h1>
-
-    <!--<p>(Updates every second.)</p>-->
+    <h1 id="percent" style="font-family: 'Courier New', Courier, monospace;"></h1>
```

Comment:
Thanks!

Please could you move the styling into `<style>` in the header above?

Reply:
Thank you for reviewing my code.
I fixed styling codes.